packages/composite/dbaas/exoscale/postgres.yml

```
@@ -0,0 +1,176 @@
+classes:
+  - ...common
+
+parameters:
+
+  pkg.appcat.composite.exoscalepostgresql:
+    defaultCompositionRef: "${facts:cloud}.exoscalepostgresqls.appcat.vshn.io"
+
+  appcat:
+    composites:
+      "exoscalepostgresqls.appcat.vshn.io":
+        spec:
+          group: appcat.vshn.io
+          names:
+            kind: XExoscalePostgreSQL
+            plural: xexoscalepostgresqls
+          claimNames:
+            kind: ExoscalePostgreSQL
+            plural: exoscalepostgresqls
+          connectionSecretKeys:
+            - POSTGRESQL_URL
+            - POSTGRESQL_DB
+            - POSTGRESQL_HOST
+            - POSTGRESQL_PORT
+            - POSTGRESQL_USER
+            - POSTGRESQL_PASSWORD
+            - ca.crt
+          defaultCompositionRef:
+            name: ${pkg.appcat.composite.exoscalepostgresql:defaultCompositionRef}
```

From #32  this should be hardcoded
```suggestion
            name: exoscale.postgres.appcat.vshn.io
```